### PR TITLE
74 add validation info

### PR DESCRIPTION
**Pull request by ignasposka**
Actual validation in resolvers will be added in another PR.

### CODE REVIEW

File: database/1-schema.sql [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 CREATE TABLE organization (
     id SERIAL PRIMARY KEY,
-    name VARCHAR(256) NOT NULL,
+    name VARCHAR(255) NOT NULL,
```
Comment (JustasBelevicius) [inline database/1-schema.sql:5]: 256 is a power of 2, not sure why the change by 1 char?

**Review by JustasBelevicius — APPROVED** [review]

File: database/1-schema.sql (hunk 1 above)
Comment (ignasposka) [reply database/1-schema.sql:5]: DB for storing input length would use 1 byte in this case. While 256 it would need to consume 2 bytes.
